odl/contrib/README.md

```diff
@@ -0,0 +1,85 @@
+# The `odl.contrib` sub-package
+
+This is a special place for user-contributed modules. Its intention is to provide a "fast lane" for contributors to include code that is
+
+- outside the main ODL library,
+- experimental and untested,
+- unfinished or still subject to major breaking changes or
+- "not quite there yet" in terms of documentation, testing or completeness.
+
+The general idea is that users get their code quickly merged without having to go through lengthy code reviews.
+
+**Note:** If you consider your code to be useful for the main library, please consider filing a pull request anyway. The maintainers try to be as helpful as possible in their reviews and make suggestions, so that getting the code ready for merging doesn't take too long.
+
+Here are some guidelines for contributors regarding the scope and the rules of this platform.
+
+
+## Scope
+
+**Yes:**
+
+- Generic code that would fit into main ODL but is still experimental
+- Special-purpose code that is not generic enough to be a fit for main ODL (e.g. [figures of merit](https://github.com/odlgroup/odl/pull/1018) or application-specific code)
```

Comment:
I guess we could do that on both of those, but lets do that after this one is in.